app/policies/rule_policy.rb

```diff
@@ -2,18 +2,18 @@
 
 class RulePolicy < ApplicationPolicy
   def index?
-    staff?
+    role.can?(:manage_rules)
```

Comment:
Those who have the `manage_reports` permission can also see the rules on the report view, plus on the public page. Given that rules are public, it doesn't make sense to add a `view_rules` flag, since everyone should have it then, but then everyone would have access to one administration menu item.

Reply:
hm… I'm not sure about it, but it's not a big deal and can always be addressed later